ansible_base/jwt_consumer/hub/auth.py

```diff
@@ -27,8 +29,10 @@ def process_permissions(self):
         self.team_content_type = ContentType.objects.get_for_model(Team)
         self.org_content_type = ContentType.objects.get_for_model(Organization)
 
-        orgs = []
-        teams = []
+        admin_orgs = []
+        member_orgs = []
+        admin_teams = []
```

Comment:
Nothing is done with the top 3 relationship right now, right? Maybe a TODO comment would be good.